packages/ckeditor5-build-helpjuice-v2/src/hjeditor.ts

```diff
@@ -349,6 +349,18 @@ HelpjuiceEditor.defaultConfig = {
 						`<video controls="" style="max-width: 100%" src="${getUrl}">Your browser does not support HTML5 video.</video>`
 					);
 				}
+			},
+			{
+				name: 'loomProvider',
+				url: new RegExp(`^https:\\/\\/www\\.loom\\.com\\/share\\/(\\w+)`),
+				html: match => {
+					const getUrl = match.input;
+					const videoId = getUrl?.split('/').pop();
```

Comment:
Instead of splitting the URL, we can extract the result from match:

```suggestion
					const videoId = match[1];
```

CKEditor docs: https://ckeditor.com/docs/ckeditor5/latest/api/module_media-embed_mediaembedconfig-MediaEmbedProvider.html
MDN docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match.

Reply:
Yes, I also just found out, will update it